node/cmd/main.go

```diff
@@ -25,6 +25,14 @@ import (
 	"github.com/ibm/ibm-block-csi-driver/node/pkg/driver"
 )
 
+func printVersion(configFile *string) {
```

Comment:
looks more like `printVersionInfo` to me (same for log message)

Reply:
Done